index.html

```diff
@@ -44,8 +44,8 @@ <h2 class="title">I'm a Software Developer</h2>
             <img class="imgf1" src="img/V12 (2).png">
             <img class="imgf2" src="img/V12 (2).png">
             <img class="imgf3" src="img/V12 (2).png">
-            <img class="imgg" src="img/V13.png">
-            <img class="imgh" src="img/V13.png">
+            <img class="imgg" src="img/v13.png">
+            <img class="imgh" src="img/v13.png">
```

Comment:
- [ ] Good work @Uchenna-ibute, you have corrected the image name as the previous reviewer(me) mentioned in the previous review, however, it is the only required change you have fixed in all those I gave, consider making all the changes as requested before you submit for your last review. Remember, according to the review policy, you can only submit 3times a PR, and this is your second one then consider making all the necessary required changes before you submit your last review, there you will have more chances not to result in a block repetition, we don't all need that. Please read again the previous review and fix them all. Good luck 🥰